Add missing url scheme in authentication sample

dpaulino opened the pull request:
In the Azure.Mobile.Client v4.x API, the LoginAsync method does not contain an overload that only takes in one parameter.

azurecla commented:
Hi __@jaymathe__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jaymathe). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

dpaulino committed:
Add missing url scheme in authentication sample

In the Azure.Mobile.Client v4.x API, the LoginAsync method does not contain a overload that only takes in one parameter.
